src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/service/AssessmentService.kt

```diff
@@ -325,4 +357,64 @@ class AssessmentService(
       policy = policy,
     )
   }
+
+  @Transactional
+  fun getContacts(prisonNumber: String): AssessmentContactsResponse {
+    val currentAssessment = this.getCurrentAssessment(prisonNumber)
+
+    // Currently we do not support Prison offender manager
+    val contactTypes = listOf(UserRole.PRISON_CA, UserRole.PRISON_DM, UserRole.PROBATION_COM)
+    val agents = getAgents(currentAssessment, contactTypes)
+
+    val contacts = agents.map {
+      with(it) {
+        var email: String? = null
+        try {
+          email = getEmailAddress(it)
+        } catch (e: ItemNotFoundException) {
+          log.debug("Could not find email with given username and role {}", it.role, e)
+        }
+
+        var location: String? = null
+        try {
+          location = getLocationName(it)
+        } catch (e: ItemNotFoundException) {
+          log.debug("Could not find Location with given username and role {}", it.role, e)
```

Comment:
Should this be a warn - all prisons should have an email?

Reply:
probably more appropriate.